### DISCUSSION TITLE
封面和目录的相关字体字号和格式调整

### DISCUSSION THREAD

**Discussion by jqc25z** [sjtug/SJTUThesis#991] [提问]
1.封面的小标题内容经过修改后的latex版本是这样的
![1701169639009](https://github.com/sjtug/SJTUThesis/assets/112072840/f12a6e78-4c84-470a-9e1d-6291c6323fbc)
x和word模板还有差距
![1701169683193](https://github.com/sjtug/SJTUThesis/assets/112072840/d08ed621-3560-44ae-ad70-75d046b8b26d)
为了显示“申请学位层次”把宽度拉宽后影响到了原先的几行。还有最后的日期字体word版是宋体，latex的是time new roman。
2.英文标题页
![1701169800832(1)](https://github.com/sjtug/SJTUThesis/assets/112072840/e8a52593-c7e7-40ca-a5f8-101f34051db4)
zhe这里需要这样都加粗和冒号对齐。
3.英文abstract的标题需要是Arial 字体，目录里的abstract也是。
4.Word模板章目录字号14磅，latex模板为12磅；Word模板二级小节字号10.5磅，latex模板为12磅。

**Comment by AlexaraWu** [comment]
SJTUThesis 的开发遵循研究生院学位办和教务处发布的《撰写规范》，不以 Word 版模板为标准，望周知。
